content/en/blog/2023-11-modelix-platform-release-23.2/index.md

```diff
@@ -0,0 +1,89 @@
+---
+date: 2023-11-02
+title: "modelix platform release 23.2"
+---
+
+We are happy to announce the Autumn release of the modelix platform: **`modelix-23.2`**
+
+The version bundled in this release are:
+
+```
+modelixCoreVersion = 3.12.1
+incrementalVersion = 0.2.0
+buildToolsVersion = 1.1.0
+modelixMPSPatchVersion = 157
+```
+
+If you want to know how to use this platform in your Gradle build, check out the corresponding [Documentation](https://docs.modelix.org/modelix/main/platform/howto/usage-platform.html).
+
+- [Feature Highlights](#feature-highlights)
+  - [ModelQL](#modelql)
+  - [Model client as NPM package (alpha)](#model-client-as-npm-package-alpha)
+  - [Vue.js Bindings (alpha)](#vuejs-bindings-alpha)
+  - [MPS Model Adapters](#mps-model-adapters)
+  - [Bulk Model Synchronization Plugin (alpha)](#bulk-model-synchronization-plugin-alpha)
+- [Development Insights](#development-insights)
+  - [New approach to MPS related components](#new-approach-to-mps-related-components)
+  - [Contribution guide](#contribution-guide)
+  - [Marking experimental features](#marking-experimental-features)
+  - [Work on code quality](#work-on-code-quality)
+- [What's to come](#whats-to-come)
+
+# Feature Highlights
+
+## ModelQL
+
+The [ModelQL query language](https://docs.modelix.org/modelix/modelix-23.2/core/explanation/modelql.html) provides a dynamic way of loading parts of the model on demand while reducing the number of requests to a minimum.
+
+## Model client as NPM package (alpha)
+
+The previously existing [model client](https://github.com/modelix/modelix.core/tree/modelix-23.2/model-client) is now also published as [`@modelix/model-client` NPM package](https://artifacts.itemis.cloud/service/rest/repository/browse/npm-open/%40modelix/model-client/).
+
+The exposed API is minimal and experimental. In the future, we want to make more features from the Kotlin model client library directly accessible. 
+
+## Vue.js Bindings (alpha)
+
+Built upon [`@modelix/model-client`](https://artifacts.itemis.cloud/service/rest/repository/browse/npm-open/%40modelix/model-client/) we created a library to interact with models through Vue.js.
+
+With [`@modelix/vue-model-api`](https://docs.modelix.org/modelix/modelix-23.2/core/reference/component-vue-model-api.html) we are currently building sample projects with Vue.js. After that, we want to stabilize the API and show you how you can use it for your projects.
+
+## MPS Model Adapters
+We created [adapters](https://github.com/modelix/modelix.core/tree/modelix-23.2/mps-model-adapters) that serve as an interface between MPS and the modelix model-api.
+They are compatible with all MPS versions that modelix currently supports.
```

Comment:
Is it really worth mentioning this in the blog post? This is for most users only an internal detail of modelix.

Reply:
I think it might be interesting for some users, so I would rather leave it in.

Reply:
Those not knowing what that is are left a bit puzzled. Maybe we can then add one sentence explaining for which purpose these adapters are useful?

Reply:
Done. I hope the added description makes it more understandable.

@mhuster23 Please check if my description is accurate.

Reply:
Left some comments